### PR TITLE
Documentation improvements 

### PR DESCRIPTION
**Pull request by roibWix**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6412

File: src/CardGalleryItem/CardGalleryItem.uni.driver.js [hunk 1]
```diff
@@ -25,34 +25,86 @@ const getSecondaryAction = async base => {
...
       ? settingsMenu._prop('firstChild')
       : null;
   },
+  /**
+   * Gets the background image element.
+   * @returns {Promise<any>}
+   */
   getBackgroundImageNode: async () => getOverlayDriver(base).getMediaNode(),
+  /**
+   * Trigger hover state on the element.
```
Comment (EgoziE) [inline src/CardGalleryItem/CardGalleryItem.uni.driver.js:105]: `Triggers` is for events and callbacks

```suggestion
   * Hovers the component.
```

File: src/CardGalleryItem/CardGalleryItem.uni.driver.js [hunk 2]
```diff
@@ -25,34 +25,86 @@ const getSecondaryAction = async base => {
...
+   * @returns {Promise<void>}
+   */
   click: async () => (await getPrimaryAction(base)).click(),
+  /**
+   * Gets the primary action label.
+   * @returns {Promise<string>}
+   */
   getPrimaryActionLabel: async () => (await getPrimaryAction(base)).text(),
+  /**
+   * Return a boolean for the primary action clickability.
```
Comment (EgoziE) [inline src/CardGalleryItem/CardGalleryItem.uni.driver.js:68]: For boolean return value, the description should be:
"Checks whether xxx"

File: src/Checkbox/Checkbox.uni.driver.js [hunk 3]
```diff
@@ -23,6 +23,10 @@ export const checkboxUniDriverFactory = (base, body) => {
 
   return {
     ...baseUniDriverFactory(base),
+    /**
+     * Click on the component root element.
```
Comment (EgoziE) [inline src/Checkbox/Checkbox.uni.driver.js:27]: We try to follow Google guides for documentation.
This is a very long read, this is the relevant section:
https://developers.google.com/style/api-reference-comments

Bottom line "Use present tense for all descriptions."

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/CircularProgressBar/docs/index.story.js [hunk 4]
```diff
@@ -78,31 +78,31 @@ export default {
...
 
-          code({
+          example({
             title: 'Themes',
             description: 'Appear in all different themes.',
             source: examples.themes,
           }),
 
-          code({
+          example({
```
Comment (TalZa1810) [inline src/CircularProgressBar/docs/index.story.js:105]: If you want to know more about the sections API, you can read [here](https://github.com/wix/wix-ui/blob/master/packages/wix-storybook-utils/docs/sections.md#sections).

File: src/ColorPicker/ColorPicker.js [hunk 5]
```diff
@@ -118,18 +118,29 @@ class ColorPicker extends React.PureComponent {
       this.setState({ current: _color });
     }
   }
-
+  /**
+   * sets the selected color
+   * @param {_color}
```
Comment (EgoziE) [inline src/ColorPicker/ColorPicker.js:123]: The pattern of @param is:

@param {\<TYPE\>} \<NAME\> \<DESCRIPTION\>

File: src/ColorPicker/ColorPicker.js [hunk 6]
```diff
@@ -118,18 +118,29 @@ class ColorPicker extends React.PureComponent {
...
+   * @returns {Void}
```
Comment (EgoziE) [inline src/ColorPicker/ColorPicker.js:124]: If the function returns void, remove the @returns statement

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Review by EgoziE — APPROVED** [review]

**Pull request merged**